README.md

```diff
@@ -213,6 +213,31 @@ type FrameMetadataResponse = {
 <br />
 <br />
 
+## Identity Kit 👤
+
+### OnchainName
+
+The `OnchainName` component primarily focuses on showcasing ENS names for given Ethereum addresses, and defaults to displaying a sliced version of the address when an ENS name isn't available.
+
+```tsx
+import { OnchainName } from '@coinbase/onchainkit';
+
+<OnchainName address="0x1234567890abcdef1234567890abcdef12345678" sliced={false} />;
+```
+
+**@Param**
```

Comment:
Oh, I think we can use types going forward to better describe what's going on. Check the latest README.

Reply:
soon we might have autogen docs that will be based on type correct?

Reply:
Maybe... There is a difference between `type` information, and how to teach how to use something.

Vitepress will automate docs creation in some capacity, but the ART of teaching how to use a function is still a manual work.